### ISSUE TITLE
Selection of Items in Navigation Bar seems to be off

### ISSUE THREAD

**Issue by phystem** [mkdocs/mkdocs#886]
Go to [Mkdocs](http://www.mkdocs.org)

Click on **Building the site**

It looks like this

![image](https://cloud.githubusercontent.com/assets/11405204/14317508/352bb85a-fc26-11e5-9f87-ff20dad0a980.png)

You can see that **Getting Help** is selected instead of **Building the site**

Shouldn't it be like this

![image](https://cloud.githubusercontent.com/assets/11405204/14317457/069cc0b0-fc26-11e5-81fe-ffdc30274740.png)

**Comment by waylan** [comment]
I can confirm the same behavior. I notice that it starts off fine with the first section, but as you scroll down the page is progressively gets worse with each section. I was sure there was another issue discussing this, but I can't seem to find it.

**Comment by phystem** [comment]
I have fixed this by using the below patch

Added as an [extra_javascript](http://www.mkdocs.org/user-guide/configuration/#extra_javascript)

```Javascript
$(document).ready(function() {
    $(".nav a").on("click", function() {
        var clicked = this;
        setTimeout(function() {
            var active = $('.nav li.active a');
            active = active[active.length - 1];
            if (clicked !== active) {
                $(active).parent().removeClass("active");
                $(clicked).parent().addClass("active");
            }
        }, 50);
    });
});
```

**Comment by d0ugal** [comment]
This has been a long standing issue, this is a duplicate of #406. I would be interested in the above change as a pull request to fix this for everyone if you want to submit one.

**Comment by phystem** [comment]
@d0ugal Seems like the error is not reproducible with the mkdocs website.Have you put any patch for that?

**Comment by d0ugal** [comment]
> @d0ugal Seems like the error is not reproducible with the mkdocs website.Have you put any patch for that?

No, it is just using the released version, MkDocs 0.15.3.

**Comment by d0ugal** [comment]
I have [included](https://github.com/mkdocs/mkdocs/pull/928) your suggestion in the MkDocs theme. It will be released with 0.16. It doesn't fully solve the problem, but we are one step closer.
